### PR TITLE
Aodn notebook

### PR DESCRIPTION
**Pull request by rfonsecadasilva**
Add notebook with examples of using aodn catalog

Plots:
- lat/lon scatter coloured by Hs and time, SAR and altimetry (see utilfuncs.py)
- timeseries plot of Hs, SAR and altimetry

Other changes:
- lat/long crop filter (see filter.py) - (a)  note that although there is a spatial filter in the URL request, the retrieved dataset still includes data outside of the requested domain, thus a spatial filter is needed; and (b) note that as lat/lon are coordinates but not dimensions of the dataset, ds.sel(...) cannot be used, thus ds.where(...) had to be used and was added to crop_filter
- sort filter applied after concatenation of dataset - note that different chunks of dataset will have each sorted data, but concatenated may present unsorted data and thus needs to be sorted

### CODE REVIEW

**Review by pbranson — APPROVED** [review]
Thanks for this improvement

**Review by pbranson — APPROVED** [review]

**Review by pbranson — APPROVED** [review]
This is a useful change

**Review by pbranson — APPROVED** [review]

File: notebooks/utilfuncs.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2020 - 2021, CSIRO 
+#
+# All rights reserved.
+#
+# The full license is in the LICENSE file, distributed with this software.
+#-----------------------------------------------------------------------------
+
+def scatter(ds,color,minLon=None,minLat=None,maxLon=None,maxLat=None,fscale=10):
```
Comment (pbranson) [inline notebooks/utilfuncs.py]: I wonder if we should start a plotting.py module within rompy and put this function there. Lets do that for now

**Review by pbranson — CHANGES_REQUESTED** [review]
Thank @rfonsecadasilva, lets just move utilfuncs.py to rompy/plotting.py and import the scatter plot function from there.

File: notebooks/utilfuncs.py (hunk 1 above)
Comment (rfonsecadasilva) [reply notebooks/utilfuncs.py]: This makes sense. By the way, I have to acknowledge that the scatter plotting function was inspired by BaseGrid/plot. Thanks, @pbranson .

**Pull request merged**